system/assets/js/admin/presidecore/preside.clickable.tableRows.js

```diff
@@ -43,7 +43,7 @@
 			var $firstLink = $( this ).find( 'a.row-link:first' );
 
 			if ( !$firstLink.length ) {
-				$firstLink = $( this ).find( 'a:first' );
+				$firstLink = $( 'a', this ).not( '.formbuilder-response a' ).first();
```

Comment:
Rather than this hardcoded exception, is there a way we can assign `row-link` to the links in the response rows that we _do_ want clickable?

Reply:
Update formbuilder submission action link with suggested `row-link` class.